### PR TITLE
Adding translated in Korean by native linguist

### PR DESCRIPTION
**Pull request by yunho0130**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -137,19 +137,23 @@ For more information on how to use QISKit, tutorial examples, and other helpful
...
 Visit the [IBM Q experience community](https://quantumexperience.ng.bluemix.net/qx/community) for questions and discussions on QISKit and quantum computing more broadly. If you'd like to contribute to QISKit, please take a look at our [contribution guidelines](CONTRIBUTING.rst).
 
+## Multilanguage guide
+
+* **[Korean Translation](https://github.com/QISKit/qiskit-sdk-py/tree/master/doc/ko/README-ko.md)**, Basic guide line written in Korean. 
 
 ## Authors (alphabetical)
 
-Jim Challenger, Andrew Cross, Vincent Dwyer, Mark Everitt, Ismael Faro, Jay Gambetta, Juan Gomez, Paco Martin, Antonio Mezzacapo, Jesus Perez, Russell Rundle, Todd Tilma, John Smolin, Erick Winston, Chris Wood
+Jim Challenger, Andrew Cross, Vincent Dwyer, Mark Everitt, Ismael Faro, Jay Gambetta, Juan Gomez, Paco Martin, Antonio Mezzacapo, Jesus Perez, Russell Rundle, Todd Tilma, John Smolin, Erick Winston, Chris Wood, Yunho Maeng
```
Comment (rraymondhp) [inline README.md]: The author list should be in alphabetical order of the family name.

File: doc/ko/README-ko.md [hunk 2]
```diff
@@ -0,0 +1,147 @@
+# Quantum Information Software Kit (QISKit)
+
+[![PyPI](https://img.shields.io/pypi/v/qiskit.svg)](https://pypi.python.org/pypi/qiskit)
+[![Build Status](https://travis-ci.org/QISKit/qiskit-sdk-py.svg?branch=master)](https://travis-ci.org/QISKit/qiskit-sdk-py)
+
+**QISKit**은 [OpenQASM](https://github.com/QISKit/qiskit-openqasm)과 [IBM Q experience (QX)](https://quantumexperience.ng.bluemix.net/)에서 사용할 수 있는 SDK(software development kit)입니다. 
+
+양자컴퓨팅 프로그램을 만들고, 컴파일하고, 실행하기 위해 **QISKit**을 사용하세요. 실행에 있어 QISKit은 온라인상 접근 가능한 실제 양자 프로세서, 온라인 시뮬레이터, 로컬 시뮬레이터의 백엔드 환경을 지원합니다. 온라인 백엔드 환경에서, QIAKit은 IBM Q experience와 연결하기 위해 [python API client](https://github.com/QISKit/qiskit-api-py)를 사용합니다.
```
Comment (rraymondhp) [inline doc/ko/README-ko.md]: QIAKit --> QISKit?

File: doc/ko/README-ko.md [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+QISKit은 본래 [IBM Research](http://www.research.ibm.com/)연구팀과 [IBM-Q](http://www.research.ibm.com/ibm-q/)개발팀에 의해 양자컴퓨터에 대한 고수준(high level) 개발킷을 제공할 목적으로 개발되었습니다. 
+질의응답과 토론 그리고 양자컴퓨팅에 대해 더 넓게 살펴보기 위해 [IBM Q experience community](https://quantumexperience.ng.bluemix.net/qx/community)를 방문하세요. 만일 여러분이 QISKit에 기여하길 원한다면 다음의 가이드라인을 살펴보세요. [contribution guidelines](CONTRIBUTING.rst).
+
+## Multilanguage guide
+
+* **[Korean Translation](https://github.com/QISKit/qiskit-sdk-py/tree/master/doc/ko/README-ko.md)**, 한글 기본 가이드 라인
+
+## Authors (alphabetical)
+
+Jim Challenger, Andrew Cross, Vincent Dwyer, Mark Everitt, Ismael Faro, Jay Gambetta, Juan Gomez, Paco Martin, Antonio Mezzacapo, Jesus Perez, Russell Rundle, Todd Tilma, John Smolin, Erick Winston, Chris Wood, Yunho Maeng
```
Comment (rraymondhp) [inline doc/ko/README-ko.md]: The authors should be sorted alphabetically of their family names

File: README.md [hunk 4]
```diff
@@ -137,19 +137,23 @@ For more information on how to use QISKit, tutorial examples, and other helpful
...
 * **[QISkit Python API](https://github.com/QISKit/qiskit-api-py)**, an API to use the IBM Quantum Experience in Python
 
-
 QISKit was originally developed by researchers and developers on the [IBM-Q](http://www.research.ibm.com/ibm-q/) Team at [IBM Research](http://www.research.ibm.com/), with the aim of offering a high level development kit to work with quantum computers.
 
 Visit the [IBM Q experience community](https://quantumexperience.ng.bluemix.net/qx/community) for questions and discussions on QISKit and quantum computing more broadly. If you'd like to contribute to QISKit, please take a look at our [contribution guidelines](CONTRIBUTING.rst).
 
+## Multilanguage guide
+
+* **[Korean Translation](https://github.com/QISKit/qiskit-sdk-py/tree/master/doc/ko/README-ko.md)**, Basic guide line written in Korean.
```
Comment (diego-plan9) [inline README.md]: Other than the suggestions by Rudy, could you please rename the file from `doc/ko/README-ko.md` to `doc/ko/README.md` and update this link accordingly?

File: README.md (hunk 1 above)
Comment (yunho0130) [reply README.md]: @rraymondhp Thanks. I changed the order by in alphabetical order of the family name.

File: doc/ko/README-ko.md (hunk 2 above)
Comment (yunho0130) [reply doc/ko/README-ko.md]: Oh.. It's my mistake. I changed it from recent commit

File: doc/ko/README-ko.md (hunk 3 above)
Comment (yunho0130) [reply doc/ko/README-ko.md]: Thanks again

File: README.md (hunk 4 above)
Comment (yunho0130) [reply README.md]: @diego-plan9 I changed file name and link accordingly.

**Comment by diego-plan9** [comment]
Thanks, @yunho0130! I have added a small fix (using a relative link instead of absolute in the main README), and it's looking good, so merging!

Please also be aware that the `README.md` is updated periodically - it would be a good idea to check back in the future eventually (for example, every major release) and adjust the Korean translation accordingly. We also have the technical documentation under `docs` that would be a good candidate for translation if you find the energy and resources :)

**Pull request merged**